src/ralph/admin/filters.py

```diff
@@ -20,7 +20,8 @@
 from ralph.admin.autocomplete import DETAIL_PARAM, QUERY_PARAM
 from ralph.admin.helpers import get_field_by_relation_path
 
-SEARCH_SEPARATORS_REGEX = re.compile(r'[;|]')
+SEARCH_OR_SEPARATORS_REGEX = re.compile(r'[;|]')
+SEARCH_AND_SEPARATORS_REGEX = re.compile(r'[,]')
```

Comment:
i'm afraid that this could be a little bit confusing (especially for users). Ex. using comma as AND separator, but semicolon as OR separator.